edx_proctoring/api.py

```diff
@@ -824,21 +824,6 @@ def update_attempt_status(exam_id, user_id, to_status,
         )
         raise ProctoredExamIllegalStatusTransition(err_msg)
 
-    # special case logic, if we are in a completed status we shouldn't allow
```

Comment:
I understand the code change here, but I think I'm missing some context around what the rationale is more generally. Could you let me know if I'm understanding this correctly?

It was the case before that a user would submit their exam and their exam attempt status would thus move to `submitted`. After that, it was possible that we would receive a `not_reviewed` status in a callback from Proctortrack, if, for example, data upload failed. Without this change, we wouldn't be able to move the exam attempt into the `error` state successfully.

Is that right?

Reply:
Correct